Mathlib.lean

```diff
@@ -3599,6 +3599,7 @@ import Mathlib.RingTheory.RingInvo
 import Mathlib.RingTheory.RootsOfUnity.Basic
 import Mathlib.RingTheory.RootsOfUnity.Complex
 import Mathlib.RingTheory.RootsOfUnity.Minpoly
+import Mathlib.RingTheory.RootsOfUnity.More
```

Comment:
I think we usually call such files `lemmas`, or something like that (of course it would be better to find a more precise name, but I understand it is not always possible).

Reply:
Changed the file name to "Lemmas.lean".